src/applications/disability-benefits/686c-674/config/constants.js

```diff
@@ -5,6 +5,18 @@ export const datePattern =
 
 export const phonePattern = '^[0-9]{10}$';
 
+export const TASK_KEYS = {
```

Comment:
Added this because we're using these strings almost everywhere. There'll be a separate ticket to update all of those references to use this constant.